src/main/frontend/persist_db/browser.cljs

```diff
@@ -112,7 +114,7 @@
                (db-transact/transact transact!
                                      (if (string? repo) repo (state/get-current-repo))
                                      tx-data
-                                     tx-meta)))
+                                     (assoc tx-meta :client-id (:client-id @state/state)))))
             (db-transact/listen-for-requests))
           (p/catch (fn [error]
                      (prn :debug "Can't init SQLite wasm")
```

Comment:
Should we delete the error handling code in this ns and in frontend.handlers.events.ui for no multi tab support e.g. :show/multiple-tabs-error-dialog?

Reply:
Yes, It's no longer needed.